docs/news.rst

```diff
@@ -46,6 +46,13 @@ Backward-incompatible changes
     so the new default should keep the old behavior. (:issue:`872`, 
     :issue:`5847`)
 
+-   The :meth:`TextResponse.json <scrapy.http.TextResponse.json>` method now 
+    has the potential to affect results when the response body lacks UTF-8 
+    encoding. This represents a backwards-incompatible change intended to 
+    improve Scrapy's efficiency in the vast majority of scenarios. However, 
+    users still have the option of using ``json.loads(response.text)`` as an 
+    alternative to the ``response.json()`` shortcut. (:issue:`5968`)
```

Comment:
You are editing the release notes of the latest, already-released version of Scrapy.

We usually don’t edit release notes on PRs, and instead do it all at once during the release process. But this is actually a good idea, to make sure we do not forget about it.

So please remove this and add the following above the 2.10.0 header (I took the liberty to make some changes):

```rst
.. _release-2.11.0:

Scrapy 2.11.0 (to be released)
------------------------------

Backward-incompatible changes
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

-   The :meth:`TextResponse.json <scrapy.http.TextResponse.json>` method now
    requires the response to be in a valid JSON encoding (UTF-8, UTF-16, or
    UTF-32).

    If you need to deal with JSON documents in an invalid encoding, use
    ``json.loads(response.text)`` instead.

    (:issue:`5968`)
```

Reply:
Thanks! The release notes are already updated